### ISSUE TITLE
Use newer/unimplemented API functionality?

### ISSUE THREAD

**Issue by qqilihq** [kklisura/chrome-devtools-java-client#23]
Is there an easy way to use an API which is currently not implemented in this library?

I'm interested in the `Overlay.highlightNode` but would like to call it with a `selector` instead of an ID. According to the [documentation](https://chromedevtools.github.io/devtools-protocol/tot/Overlay#method-highlightNode) this is possible with the most recent version, but this library currently does not allow to give a selector:

<img width="927" alt="image" src="https://user-images.githubusercontent.com/2172260/63761871-20bd5380-c8c2-11e9-8895-62371df0ed9c.png">

If it makes sense I could alternatively see whether I can provide a PR.

PS: Thanks for this, it's very handy for me!

**Comment by qqilihq** [comment]
I figured this out (and I probably look quite lazy now, sorry). There are two options:

1. One way would be to re-generated the code using the [Java Protocol Builder](https://github.com/kklisura/chrome-devtools-java-client/tree/master/cdt-java-protocol-builder)

2. Alternatively, use the `com.github.kklisura.cdt.services.ChromeDevToolsService.invoke(String, Class<Void>, MethodInvocation)` method directly without an interface. This would look as follows:

   ```
   MethodInvocation methodInvocation = new MethodInvocation();
   methodInvocation.setId(someId);
   methodInvocation.setMethod("Overlay.highlightNode");
   Map<String, Object> map = new HashMap<>();
   map.put("backendNodeId", backendNodeId);
   map.put("highlightConfig", highlightConfig);
   map.put("selector", selector);
   methodInvocation.setParams(map);
   devToolsService.invoke(null, Void.class, methodInvocation);
   ```

**Comment by kklisura** [comment]
Hi, thanks for using this library. I've just published `v2.0.0` with the updated protocol. I've also included an example on how to highlight element https://github.com/kklisura/chrome-devtools-java-client/blob/master/cdt-examples/src/main/java/com/github/kklisura/cdt/examples/HighlightElementExample.java. :)

**Comment by qqilihq** [comment]
Hi there! Awesome -- thank you very much!
